experimental/dds/tree/src/persisted-types/0.1.1.ts

```diff
@@ -481,6 +481,9 @@ export const reservedIdCount = 10;
 // This UUID must never change
 export const ghostSessionId = '79590933-1c70-4fda-817a-adab57c20318' as SessionId;
 
+/** The SessionId of the temporary IdCompressor that records stashed ops */
+export const stashedSessionId = '8477b8d5-cf6c-4673-8345-8f076a8f9bc6' as SessionId;
```

Comment:
this uuid and ghostSessionId are never (and should never) be persisted; if they were it'd be a huge bug due to collisions. we should move these out into some other private utility file.

Reply:
The `ghostSessionId` actually is persisted, since it's the attribution for all pre-upgrade IDs. You're right about `stashedSessionId`, that should just be a local in SharedTree.ts; it's totally internal.